### PR TITLE
EGATE-58: Removed Events dependencies as we moved alfresco-events-rep…

### PR DESCRIPTION
**Pull request by jamalkm**
EGATE-35/EGATE-58: Removed Events dependencies as we moved 'alfresco-events-repo' and 'alfresco-messaging-repo' to alfresco-repository project. Also, there is no need for 'alfresco-events-repo-activiti' and 'alfresco-events-repo-publishing' as they were only required for Analytics.

Note: There is a build failure: https://bamboo.alfresco.com/bamboo/browse/PLAT-ACS46-3 but I'm not sure if it is related to these changes or not as locally everything works as expected.

### CODE REVIEW

**Comment by ancutam** [comment]
The job that is failing in the build, `All Amps validation`, applies a series of amps on alfresco, `sync-services` amp being one of those.
@jamalkm Since you removed the dependency on alfresco-events, but didn't update the alfresco-repository to a version which contains the events code, I believe that is causing the failure.
So before merging this change here to master, please do a release from the feature branch on alfresco-repository and update it in acs-packaging.

**Comment by jamalkm** [comment]
@ancutam as far as I know, sync service amp still doesn't work with ACS6. I thought this amp was disabled.

**Comment by ancutam** [comment]
@jamalkm Sync Services **3.0-RC1** is compatible with ACS 6 https://github.com/Alfresco/acs-packaging/blob/master/pom.xml#L99.
And is listed through the supported components https://github.com/Alfresco/acs-packaging/blob/master/distribution/src/main/resources/VERSIONS.md

**Comment by jamalkm** [comment]
Ok, the build is green now. We need to release another RC for Sync service as well. So, first we need to:
1. merge alfresco-repository EGATE-55 branch to master and release
2. release Sync service 3.0-RCx
3. update pom.xml file of this project with the released versions

**Comment by derekhulley** [comment]
@jamalkm You need to do the sync work against Sync 3.1-SNAPSHOT because you are generating a sync that is not against the 6.0 version of the repo.  In fact, if you merge to master of the repository just now, you run the risk of introducing events into the repo just as we go into releasing ACS 6.0.
Please hold off any actual merging to master for this work as we need to get 6.0 out.

**Comment by jamalkm** [comment]
Updated:
alfresco-repository to **7.4**
alfresco-enterprise-repository to **7.1**
alfresco-remote-api to **7.0**
alfresco-enterprise-remote-api to **7.0**

sync-service to **3.1-EGATE55**

File: pom.xml [hunk 1]
```diff
@@ -98,7 +96,7 @@
         <alfresco.mm.version>1.2.1</alfresco.mm.version>
 
         <!-- Alfresco Desktop Sync Service Module -->
-        <alfresco.desktop-sync.version>3.0</alfresco.desktop-sync.version>
+        <alfresco.desktop-sync.version>3.1-EGATE55</alfresco.desktop-sync.version>
```
Comment (ancutam) [inline pom.xml]: Is this still the expected version for the sync services module?

Comment (jamalkm) [reply pom.xml]: Yes, they are working on 3.0.1 bug-fix and they don't know when they can release 3.1.

Comment (derekhulley) [reply pom.xml]: Ideally we'd get dsync 3.1-RC1 ... if not, please confirm that the tag created will not be deleted.

**Comment by jamalkm** [comment]
The 2nd build is green too. I'm going to merge it now.

**Pull request merged**